cmd/kor/exporter.go

```diff
@@ -0,0 +1,21 @@
+package kor
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/yonahd/kor/pkg/kor"
+)
+
+var exporterCmd = &cobra.Command{
+	Use:   "exporter",
```

Comment:
Wouldn't it be better if this command specified the destination where it's exporting to? I mean maybe in the future some other feature could try to export somewhere else other then to Prometheus.
What do you think of having something like:

`kor exporter --prometheus`
or
`kor prom-exporter`

Reply:
sounds good, but since we only have Prometheus support no need for a change
we can keep it the default unless specified other, same as in the output format .. 
WDYT ?